src/serlio/scripts/serlioCreateUI.mel

```diff
@@ -60,6 +60,37 @@ global proc createPrtNode() {
 	select -r $select;
 }
 
+global proc removePrtNode() {
+	string $select[] = `ls -sl`;
+	string $initialShapes[] = collectInitialShapes($select);
+	
+	if(!(size($initialShapes))) {
```

Comment:
for readability, please use `== 0` notation for zero size checks

Reply:
changed, also changed a few other instances with the same issue.